src/goto-programs/system_library_symbols.cpp

```diff
@@ -106,7 +106,7 @@ void system_library_symbolst::init_system_library_map()
   std::list<irep_idt> setjmp_syms=
   {
     "_longjmp", "_setjmp", "longjmp", "longjmperror", "setjmp",
-    "siglongjmp", "sigsetjmp"
+    "siglongjmp", "sigsetjmp", "jmp_buf"
```

Comment:
Nit pick: please keep this lexicographically sorted, even though this increases the code churn.

Reply:
done